### PR TITLE
Bug 1827638: Rename CRD to AWSEBSDriver

### PR DESCRIPTION
**Pull request by jsafrane**
And move to "csi.openshift.io" API group.
Fixes: #47 
@openshift/storage 

Go code won't compile if there is a typo, so please review especially the yaml file changes carefully.

### CODE REVIEW

File: pkg/apis/operator/v1alpha1/doc.go [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // +k8s:deepcopy-gen=package
-// +groupName=ebs.aws.csi.openshift.io
+// +groupName=csi.openshift.io
 
 // Package v1alpha1 is the v1alpha1 version of the API.
 package v1alpha1 // import "github.com/aws-ebs-csi-driver-operator/pkg/apis/operator/v1alpha1"
```
Comment (bertinatto) [inline pkg/apis/operator/v1alpha1/doc.go:5]: If you happen to need to add more changes to this PR, could you add *openshift* to this URL?

Comment (jsafrane) [reply pkg/apis/operator/v1alpha1/doc.go:5]: fixed

File: pkg/operator/operator.go [hunk 2]
```diff
@@ -472,7 +472,7 @@ func addFinalizer(instance *v1alpha1.Driver, f string) bool {
 	return true
 }
 
-func removeFinalizer(instance *v1alpha1.Driver, f string) {
+func removeFinalizer(instance *v1alpha1.AWSEBSDriver, f string) {
```
Comment (bertinatto) [inline pkg/operator/operator.go:475]: nit: the finalizer is called `csi.storage.operator.openshift.io`

Comment (jsafrane) [reply pkg/operator/operator.go:475]: Fixes (as a separate commit)

**Pull request merged**